sycl/source/builtins_generator.py

```diff
@@ -1339,6 +1339,11 @@ def generate_file(directory, file_name, extra_includes, generated_builtins):
   """Generates a builtins header."""
   instantiated_extra_includes = ('\n'.join([f'#include <{inc}>' for inc in extra_includes]))
 
+  if file_name == 'builtins_scalar_gen.hpp':
+      include = 'sycl/builtins_utils_scalar.hpp'
+  else:
+      include = 'sycl/builtins_utils_vec.hpp'
```

Comment:
I would prefer we add these under `extra_includes` instead of making special rules by `file_name` outside where it is set.

Reply:
Please check the last commit.